dags/iop/dag_pull_ftp.py

```diff
@@ -19,6 +19,16 @@ def iop_pull_ftp():
 
     @task()
     def migrate_from_ftp(repo=IOPRepository(), sftp=IOPSFTPService(), **kwargs):
+        params = kwargs["params"]
+        specific_files = (
+            "filenames_pull" in params
+            and params["filenames_pull"]["enabled"]
+            and not params["filenames_pull"]["force_from_ftp"]
```

Comment:
the keys `enabled` and `force_from_ftp` are always in params?

Reply:
yes, its has to be always in Params, if the publisher's source is SFTP/FTP